mut/stage.py

```diff
@@ -84,6 +85,33 @@
 T = TypeVar('T')
 
 
+class CacheControl:
+    """A manager for cache-control headers. When constructed it takes a list of stanzas,
+       each of the type Tuple[List[str], str]: this maps a set of shell-style wildcards
+       to a cache-control header. See the DEFAULT_PATTERN for an example.
+    """
+    # Based on https://www.gatsbyjs.com/docs/caching/
+    DEFAULT_PATTERN = '''[
+        [["*.html", "*/page-data/*.json", "*/sw.js"],
+            "public, max-age=0, must-revalidate"
+        ],
+        [["*"], "public, max-age=28800"]
```

Comment:
Gatsby suggests that all other resources can be cached forever, any reason we are still opting for 8 hours?

Reply:
These rules are applied to both legacy and gatsby

Reply:
Oh yes! Thanks for clarifying 👍